recipes/jigsawpy/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "jigsawpy" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/dengwirda/jigsaw-python/archive/v{{ version }}.tar.gz
+  sha256: 36901576cee361d8ee0d2977e9022626b37801f47aebd234f8a008c74557e711
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools
+  run:
+    - python >=3.6
+    - jigsaw >=0.9.12
+    - numpy
+    - pathlib
+
+test:
+  requires:
+    - matplotlib
+  source_files:
+    - files/*.msh
+    - tests/*.py
+    - example.py
+    - cache/.keep
+  commands:
+    - ./example.py --IDnumber 0
+    - ./example.py --IDnumber 1
+    - ./example.py --IDnumber 2
+    - ./example.py --IDnumber 3
+    - ./example.py --IDnumber 4
+    - ./example.py --IDnumber 5
+    - ./example.py --IDnumber 6
+    - ./example.py --IDnumber 7
+    - ./example.py --IDnumber 8
+
+  home: https://github.com/dengwirda/jigsaw-python
+  license: custom
+  license_file: LICENSE.md
+  summary: |
+    jigsawpy is the python interface to JIGSAW, which is a Delaunay-based
+    unstructured mesh generator for two- and three-dimensional geometries.
+  description: |
+    JIGSAW is a stand-alone mesh generator written in c++, based on the
+    general-purpose meshing package JIGSAW. This toolbox provides a Python
+    based scripting interface, including file I/O, mesh visualisation and
+    post-processing facilities. The underlying JIGSAW library is a collection
+    of unstructured triangle- and tetrahedron-based meshing algorithms,
+    designed to produce high quality Delaunay-based grids for computational
+    simulation. JIGSAW includes both Delaunay-refinement based algorithms for
+    the construction of new meshes, as well as optimisation driven methods for
+    the improvement of existing grids.
+  doc_url: https://github.com/dengwirda/jigsaw-python
+  dev_url: https://github.com/dengwirda/jigsaw-python
+
+extra:
+  recipe-maintainers:
+    - xylar
+    - dengwirda
```

Comment:
@dengwirda, can you confirm that you are willing to be a maintainer of this recipe?

Reply:
Yes, of course @xylar!